com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -197,6 +219,10 @@ internal static class ShaderPropertyId
         public static readonly int inverseCameraProjectionMatrix = Shader.PropertyToID("unity_CameraInvProjection");
         public static readonly int worldToCameraMatrix = Shader.PropertyToID("unity_WorldToCamera");
         public static readonly int cameraToWorldMatrix = Shader.PropertyToID("unity_CameraToWorld");
+
+        public static readonly int sourceTex = Shader.PropertyToID("_SourceTex");
+        public static readonly int scaleBias = Shader.PropertyToID("_ScaleBias");
+        public static readonly int scaleBiasRt = Shader.PropertyToID("_ScaleBiasRt");
```

Comment:
Rt is not very descriptive either, is this a scale bias matrix/projection?

Reply:
We are following URP copy depth pass naming convention here.
I believe it is used for scale bias render target destination viewport rect.
We can address this in the next PR.